modin/pandas/test/test_series.py

```diff
@@ -2086,19 +2088,37 @@ def test_pow(data):
     inter_df_math_helper(modin_series, pandas_series, "pow")
 
 
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_prod(data):
-    modin_series, pandas_series = create_test_series(data)
-    # Wrap in Series to test almost_equal because of overflow
-    df_equals(pd.Series([modin_series.prod()]), pandas.Series([pandas_series.prod()]))
-
-
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_product(data):
-    modin_series, pandas_series = create_test_series(data)
-    # Wrap in Series to test almost_equal because of overflow
-    df_equals(
-        pd.Series([modin_series.product()]), pandas.Series([pandas_series.product()])
+@pytest.mark.parametrize(
+    "data",
+    test_data_values + test_data_small_values,
+    ids=test_data_keys + test_data_small_keys,
+)
+@pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
+@pytest.mark.parametrize(
+    "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
+)
+@pytest.mark.parametrize(
+    "numeric_only",
+    [
+        None,
+        False,
+        pytest.param(
+            True,
+            marks=pytest.mark.xfail(reason="numeric_only not implemented for Series"),
+        ),
```

Comment:
Why `xfail`? Is modin behavior different from pandas behavior?

Reply:
`numeric_only=True` is not implemented for `pandas.Series` and throws exception, but `modin.Series` supports this parameter, should we copy such type of behavior?

Reply:
```suggestion
        pytest.param(
            True,
            marks=pytest.mark.xfail(reason="numeric_only not implemented for pandas.Series"),
        ),
```

Reply:
No, but let's reflect that the problem is not on our side.